config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
Na taketo tu uz mame gem `Figaro`, ktory automaticky nacita `application.yml`. Je tu vsak zadrhel v tom, ze kedze sa to aktualne pouziva na secret kluce, tak je to v codebase len ako `.yml.example`, ktory si treba vykopirovat a upravit. Cize:
1. Pridat tie premenne do `application.yml.example` a pridat krok do README, ze podobne ako si treba spravit `database.yml` zalozene na `.example`, tak si bude treba spravit aj `application.yml`. Nevyhoda: ked sa zmenia tieto vahy, musi si to upravit kazdy sam a treba to upravit na serveri.
alebo:
2. Neist vobec cez ENV. Toto podla mna skor nie su veci, ktore sa tykaju daneho nasadenia (ako secret kluce, databazove hesla a pod.), ale spravit konstanty na vhodnom mieste (nejaky `lib`, etc.).

@petergaspar Comments?